src/root/views/RegionalRiskWatch/ImminentEvents/ImminentEventsGDACS/index.tsx

```diff
@@ -96,6 +98,13 @@ function ImminentEventsGDACS(props: Props) {
     url: `risk://api/v1/gdacs/${eventId}/exposure`,
   });
 
+  const regionBounds = React.useMemo(
+    () => {
+      let bbox = turfBbox(region?.bbox ?? []);
```

Comment:
Can we even pass empty array on turfBbox function?
Why are we passing bbox on turfBbox?